Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -4199,13 +4204,15 @@ def add_feed_tags_command(client: Client, args: dict[str, Any]) -> CommandResult
     Returns:
         CommandResults: JSON response of added tags.
     """
-    result = client.add_feed_tags(args)
+    result = client.add_feed_tags(args).get('created_or_updated')
+    uuid = args.get('feed_uuid')
+    tags = args.get('tags')
 
     return CommandResults(
         outputs_prefix="SilentPush.AddFeedTags",
         outputs_key_field="feed_uuid",
         outputs=result,
-        readable_output=tableToMarkdown("SilentPush Add Feed Tags", result),
+        readable_output=tableToMarkdown(f"feed with uuid: {uuid} was updated with tags: {tags}", result),
```

Comment:
no need to return the feeds with the tags, the string alone is enough:
readable_output=f"feed with uuid: {uuid} was updated with tags: {tags}"